DEPLOYMENTS.md

```diff
@@ -85,7 +85,7 @@
 | `EtherspotPaymaster` | `chiado` | [0x898c530A5fA37720DcF1843AeCC34b6B0cBaEB8a](https://blockscout.chiadochain.net/address/0x898c530A5fA37720DcF1843AeCC34b6B0cBaEB8a) | [0x94f18bf16505e8064fbee8f58361576109171b3f7fe1ef5b81fb61b5fb722e1a](https://blockscout.chiadochain.net/tx/0x94f18bf16505e8064fbee8f58361576109171b3f7fe1ef5b81fb61b5fb722e1a) |
 | `EtherspotWalletFactory` | `chiado` | [0x7f6d8F107fE8551160BD5351d5F1514A6aD5d40E](https://blockscout.chiadochain.net/address/0x7f6d8F107fE8551160BD5351d5F1514A6aD5d40E) | [0x3f11f840a1c379baf5a63d12edb3ffabf2a21d16cf55c51211c43fde01e47f2b](https://blockscout.chiadochain.net/tx/0x3f11f840a1c379baf5a63d12edb3ffabf2a21d16cf55c51211c43fde01e47f2b) |
 | `EtherspotWallet` | `chiado` | [0xfB32cef50CfB0A0F9f6d37A05828b2F56EfdfE20](https://blockscout.chiadochain.net/address/0xfB32cef50CfB0A0F9f6d37A05828b2F56EfdfE20) | [0x6313bb56809aa3b51d9bf682d57ce4b39297bc897bced92af1bdadd902a7d221](https://blockscout.chiadochain.net/tx/0x6313bb56809aa3b51d9bf682d57ce4b39297bc897bced92af1bdadd902a7d221) |
-| `EtherspotPaymaster` | `rootstockTestnet` | [0x898c530A5fA37720DcF1843AeCC34b6B0cBaEB8a](https://explorer.testnet.rsk.co/address/0x898c530A5fA37720DcF1843AeCC34b6B0cBaEB8a) | [0x46d1bb68df9450c81a9814ed95cd58ee87ae290c9f13e06752f5a362fb192508](https://explorer.testnet.rsk.co/tx/0x46d1bb68df9450c81a9814ed95cd58ee87ae290c9f13e06752f5a362fb192508) |
+| `EtherspotPaymaster` | `rootstockTestnet` | [0xD302BE6e9D3fE0fBf8122a0C34611E31c0D0E792](https://explorer.testnet.rsk.co/address/0xD302BE6e9D3fE0fBf8122a0C34611E31c0D0E792) | [0x46d1bb68df9450c81a9814ed95cd58ee87ae290c9f13e06752f5a362fb192508](https://explorer.testnet.rsk.co/tx/0x46d1bb68df9450c81a9814ed95cd58ee87ae290c9f13e06752f5a362fb192508) |
```

Comment:
Could you please replace explorer links with https://rootstock-testnet.blockscout.com/ and also for rootstock mainnet to https://rootstock.blockscout.com/

Reply:
Done